src/ui/src/js/configs/dt_renderer.js

```diff
@@ -26,42 +26,42 @@ export default function runDTRenderer(DTRendererService) {
             $('<label>')
               .attr('for', 'childCheck')
               .css('padding-left', '4px')
-              .text('Include Children')
+              .text('Show Children')
```

Comment:
You need to update the attribute here to 'filterHidden' to match what you did on line 61

Reply:
No, this label is `for` the input defined above it, and the id of that input is `childCheck`.

These blocks are all independent. I wish we could consolidate them into one so we wouldn't have to worry about this, but I haven't figure out how to do that.